### PR TITLE
Push nginx-1.20 container to quay.io/fedora/nginx-120

### PR DESCRIPTION
**Pull request by phracek**

### CODE REVIEW

**Comment by phracek** [comment]
[test]

**Comment by phracek** [comment]
[test]

**Comment by phracek** [comment]
Let's test Fedora tests on Fedora host.

[test]

File: README.md [hunk 1]
```diff
@@ -3,10 +3,6 @@ Nginx container images
 
 [![Build and push container images to quay.io registry](https://github.com/sclorg/nginx-container/actions/workflows/build-and-push.yml/badge.svg)](https://github.com/sclorg/nginx-container/actions/workflows/build-and-push.yml)
 
-nginx-container 1.18 status: [![Docker Repository on Quay](https://quay.io/repository/centos7/nginx-118-centos7/status "Docker Repository on Quay")](https://quay.io/repository/centos7/nginx-118-centos7)
```
Comment (zmiklank) [inline README.md:6]: Could you please add info about building and publishing fedora images to the readme?

Comment (phracek) [reply README.md:6]: I will look at it, but basically, If build-and-push is working and it is green, then everything, like C8S, C9S, Fedora, and CentOS 7 is working properly, but reference to the images is fine. The status will not be ok, I will change it somehow.

Comment (phracek) [reply README.md:6]: Fixed by https://github.com/sclorg/nginx-container/pull/183/commits/3f7a254a89a8eae7fa81039046ab8b484a5ee5b4

**Comment by phracek** [comment]
[test-all]

File: README.md (hunk 1 above)
Comment (zmiklank) [reply README.md:6]: There is also line:
`This repository contains Dockerfiles for Nginx images for OpenShift. Users can choose between RHEL and CentOS based images.` which is no longer true, and
```
Versions

Nginx versions currently provided are:

    [nginx-1.18](https://github.com/sclorg/nginx-container/blob/3f7a254a89a8eae7fa81039046ab8b484a5ee5b4/1.18)
    [nginx-1.20](https://github.com/sclorg/nginx-container/blob/3f7a254a89a8eae7fa81039046ab8b484a5ee5b4/1.20)

RHEL versions currently supported are:

    RHEL7
    RHEL8

CentOS versions currently supported are:

    CentOS7
    CentOS Stream 8
    CentOS Stream 9
```
where Fedora should also be added.

Comment (phracek) [reply README.md:6]: I will update the row in OpenShift use case, but among all sclorg repositories, `Fedora versions currently supported are:` is not mentioned at all.

Comment (phracek) [reply README.md:6]: I would prefer to update documentation as a separate Pull Request. But it brings one thing to my mind. Do we want to review the documentation among all repositories?

Comment (zmiklank) [reply README.md:6]: We probably should review it.

**Comment by phracek** [comment]
[test-all]

File: 1.18/README.md [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
 ============================================================
 This container image includes Nginx 1.18 server and a reverse server for OpenShift and general usage.
-Users can choose between RHEL, CentOS and Fedora based images.
+Users can choose between RHEL, CentOS, CentOS Stream and Fedora based images.
 The RHEL images are available in the [Red Hat Container Catalog](https://access.redhat.com/containers/),
-the CentOS images are available on [Quay.io](https://quay.io/organization/centos7),
-and the Fedora images are available in [Fedora Registry](https://registry.fedoraproject.org/).
+the CentOS images are available in the [Quay.io](https://quay.io/organization/centos7),
+the CentOS Stream images are available in the [Quay.io](https://quay.io/organization/sclorg),
+and the Fedora images are available in the [Quay.io](https://quay.io/organization/fedora).
```
[the remark this answers is not in the document]
Comment (phracek) [reply 1.18/README.md:8]: This is a good point, but among all repositories, we use the general link, where the images live. I would prefer to add a link and use can browse even all the other images.

[the remark this answers is not in the document]
Comment (zmiklank) [reply 1.18/README.md:8]: Ok.

**Review by zmiklank — APPROVED** [review]

**Pull request merged**